src/core/plugins/deep-linking/expScr.js

```diff
@@ -1,16 +1,14 @@
-import zenscroll from "zenscroll"
+import scrollToElement from "scroll-to-element"
```

Comment:
Any reason this was done? We moved to `zenscroll` from `scroll-to-element` in December because Zenscroll had better support for scrolling within containers (see https://github.com/swagger-api/swagger-ui/pull/3812).

That functionality needs to continue to be supported, since many projects use Swagger UI within a container.